velox/exec/tests/PrefixSortTest.cpp

```diff
@@ -57,18 +57,30 @@ class PrefixSortTest : public exec::test::OperatorTestBase {
         rowType->children().end()};
 
     RowContainer rowContainer(keyTypes, payloadTypes, pool_.get());
-    std::vector<char*> rows = storeRows(numRows, data, &rowContainer);
-
+    auto rows = storeRows(numRows, data, &rowContainer);
+    const std::shared_ptr<memory::MemoryPool> sortPool =
+        rootPool_->addLeafChild("prefixsort");
+    const auto maxBytes = PrefixSort::maxRequiredBytes(
+        sortPool.get(),
+        &rowContainer,
+        compareFlags,
+        common::PrefixSortConfig{
+            1024,
+            // Set threshold to 0 to enable prefix-sort in small dataset.
+            0});
+    const auto beforeBytes = sortPool->peakBytes();
```

Comment:
You shall check used bytes instead of peak bytes here

Reply:
The usedBytes is always 0, because `allocs 12, frees 2,`